### PR TITLE
Print out the rsync command (including parameters)

### PR DESCRIPTION
**Pull request by adrianreber**
For better problem analysis it is helpful to know
which rsync command is actually used by MirrorManager.

### CODE REVIEW

**Comment by adrianreber** [comment]
This is something which I was always looking for. Which rsync command is actually used by MirrorManager for the crawling. I am not sure if this is this is the correct python way of doing things but it seems to work for my tests.

File: utility/mm2_update-master-directory-list [hunk 1]
```diff
@@ -622,8 +622,9 @@ def parse_rsync_listing(session, config, category, f):
 
 
 def sync_directories_using_rsync(session, config, rsyncpath, category):
+    global logger
```
Comment (ralphbean) [inline utility/mm2_update-master-directory-list]: No need for the ``global logger`` statement here.

**Comment by adrianreber** [comment]
I updated this PR and removed the ```global logger``` line.

**Comment by pypingou** [comment]
Cool, thanks!

**Pull request merged**